packages/components/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # @hashicorp/design-system-components
 
+## 2.4.0
+
+### Minor Changes
+
+- [#1371](https://github.com/hashicorp/design-system/pull/1371) [`aa2be65cf`](https://github.com/hashicorp/design-system/commit/aa2be65cf3df7742c09d772b4f14b2b56f549b53) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! - Refactor `Hds::Disclosure` internal utility component into two new components:
+  - Rename the original `Hds::Disclosure` component to `Hds::MenuPrimitive`
+  - Add a new `Hds::DisclosurePrimitive` component stripped of the “click outside/unfocus/esc to close” functionality
+
+### Patch Changes
+
+- [#1373](https://github.com/hashicorp/design-system/pull/1373) [`b2949f208`](https://github.com/hashicorp/design-system/commit/b2949f2087b0e52574a927bb095dc748a6aa81c3) Thanks [@MelSumner](https://github.com/MelSumner)! - Updates aria-label support for consistency. Consumers can now see in the component API docs where `@ariaLabel` is supported for a custom value, and what the fallback value is.
```

Comment:
@hashicorp/hds-engineering not fussy about this one, but my perception is that this is a feature, hence a minor – it will be released as part of a minor anyway, but checking others' understanding of it

Reply:
I thought so too but I got the feedback that it should be okay as a patch so that's what I went with. I will make sure to ping you next time explicitly since we seem to be on the same page about this.

Reply:
@alex-ju I told Melanie to use a patch (didn't consider this as a feature at the time, because we're not changing what the consumers see, but on a second thought you're right, we're extending the APIs of the components)

Reply:
all good in the end, just wanted to make sure we have a similar understanding 🙌